### PR TITLE
[AB-21] FE: Header component

### PR DESCRIPTION
**Pull request by twentymurial33**
**Acceptance Criteria**

- BGU should see a new header design based on Figma
- Should follow all responsive screens
- Navbar still exists and has the same functionality
- This component is a dumb/presentational component and stateless

Current State 
Screenshot 2024-01-28 at 3 41 04 PM

### CODE REVIEW

File: components/CmsHeader/index.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+import Button from "components/Common/Button";
+import {
+  CmsHeaderWrapper,
+  LinkWrapper,
+  HeaderContainer,
+  TitleContainer,
+  ArticleContainer,
+} from "./styled";
+
```
Comment (idiglove) [inline components/CmsHeader/index.tsx]: good start! proud of you!

can we have a convention of when using styled components we suffix it with Styled?
bcos i get confused if its a styled comp or a react comp :D

File: components/CmsHeader/styled.ts [hunk 2]
```diff
@@ -0,0 +1,47 @@
+import styled from "styled-components";
+
+export const CmsHeaderWrapper = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+`;
+export const LinkWrapper = styled.div`
+  width: 198px;
```
Comment (idiglove) [inline components/CmsHeader/styled.ts:9]: these are very arbitrary numbers. we should revisit this. i will create a design system for Links and Buttons. so ignore for now

File: components/Header/styled.ts [hunk 3]
```diff
@@ -226,6 +226,17 @@ export const Rubik = styled.h1`
   margin-bottom: 1rem;
 `;
 
+export const Rubikcms = styled.h1`
```
Comment (idiglove) [inline components/Header/styled.ts]: didnt know this was an h1.
you can say HeadingStyled

File: components/CmsHeader/styled.ts (hunk 2 above)
Comment (twentymurial33) [reply components/CmsHeader/styled.ts:9]: ok

File: components/Header/styled.ts (hunk 3 above)
Comment (twentymurial33) [reply components/Header/styled.ts]: renamed it to `CmsStyledHeader`

File: components/Header/styled.ts
Comment (idiglove) [inline components/Header/styled.ts:1]: lets make the convention as Styled as suffix
so it will be CMSHeaderStyled

File: assets/img/CMSALPACA.svg
Comment (idiglove) [inline assets/img/CMSALPACA.svg:1]: lets remove this file

File: components/CmsHeader/index.tsx [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+  CmsHeaderWrapperStyled,
+  LinkWrapperStyled,
+  HeaderContainerStyled,
+  TitleContainerStyled,
+  ArticleContainerStyled,
+  ArticleTextStyled,
+  AuthorTextStyled,
+  CmsHeaderStyled,
+  StyledContainer,
+} from "./styled";
```
Comment (idiglove) [inline components/CmsHeader/index.tsx:19]: very nice. pls comment on mine too if i forget to do this convention :)

File: components/CmsHeader/index.tsx [hunk 5]
```diff
@@ -0,0 +1,73 @@
+import { HeaderWrapper } from "../Header/styled";
+import Navbar from "components/Navbar";
+import Link from "next/link";
+import Tent from "assets/img/Tent";
+import arrowIcon from "../../assets/img/arrow.svg";
```
Comment (idiglove) [inline components/CmsHeader/index.tsx:5]: pls make a react component out of it, and it import it as tsx
just like assets/img/AlpacaAvatar.js

File: components/CmsHeader/index.tsx [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+            <Link style={{ marginRight: "10px" }} href="/reviews">
+              Reviews
+            </Link>
+            <Image
+              style={{ marginRight: "10px" }}
+              src={arrowIcon}
+              alt="Logo"
+              width={6}
+              height={19}
+            />
```
Comment (idiglove) [inline components/CmsHeader/index.tsx:37]: no need to use image component, refer to my comment above and use it as a react component

File: components/CmsHeader/styled.ts [hunk 7]
```diff
@@ -0,0 +1,113 @@
...
+  @media (max-width: 768px) {
+    margin-left: 0;
+  }
+`;
+export const ArticleContainerStyled = styled.div`
+  background-color: #e0e0e0;
+  height: 602px;
+  width: 435px;
+  border: 1px solid #000;
+  padding: 238px 41px 340px 42px;
```
Comment (idiglove) [inline components/CmsHeader/styled.ts:51]: very arbitrary numbers, will check

**Review by idiglove — CHANGES_REQUESTED** [review]
good work. will have to run and test

File: components/CmsHeader/styled.ts (hunk 7 above)
Comment (idiglove) [reply components/CmsHeader/styled.ts:51]: oh ok this is the featured image, pls ignore

**Review by idiglove — COMMENTED** [review]
HeaderContainerStyled should have a padding around it. 
Lets not put outer spacing on breadcrumbs or headers. Let the Container handle the outer padding

File: components/CmsHeader/index.tsx (hunk 4 above)
Comment (twentymurial33) [reply components/CmsHeader/index.tsx:19]: Thank you

File: components/CmsHeader/index.tsx (hunk 6 above)
Comment (twentymurial33) [reply components/CmsHeader/index.tsx:37]: In a later comment you told me to leave it - so wasn't sure

Comment (idiglove) [reply components/CmsHeader/index.tsx:37]: oh is it, im going nuts XD But lets fix it here. Icons ideally should be svgs. I will check this PR now and test

**Review by idiglove — APPROVED** [review]
just make the changes i said on discord.
I'll approve this now, and just merge it when youre ready

Dont forget to remove your /pages/cms file

**Pull request merged**